### DISCUSSION TITLE
Pl¢tfarmer

### DISCUSSION THREAD

**Discussion by giscus** [aduros/wasm4#547] [Games]
# Pl¢tfarmer

Build retro games using WebAssembly for a fantasy console

https://wasm4.org/play/plctfarmer/

**Comment by CanyonTurtle** [comment]
This game is a hidden gem, period. 


**Spoilers**

Don't let the farm fool you; this is a precision platformer, requiring millisecond-timed jumps and pivots. The game design is resourceful and original, using the idle farm as a puzzle mechanic, then reusing locations with routes locked behind lategame mechanics. That thoughtfulness, combined with using the Wasm4's pallete to create differrnt aesthetics in each area, make this an immersive, impressively frustrating, engaging experience.
